gtsfm/data_association/point3d_initializer.py

```diff
@@ -59,6 +59,8 @@ class Point3dInitializer(NamedTuple):
     """Class to initialize landmark points via triangulation w/ or w/o RANSAC inlier/outlier selection.
 
     Note: We currently limit the size of each sample to 2 camera views in our RANSAC scheme.
+    Comparable to OpenSfM's `TrackTriangulator` class:
+        https://github.com/mapillary/OpenSfM/blob/master/opensfm/reconstruction.py#L755
```

Comment:
Did we actually refer to OpenSfM's implementation when implementing this? Because every SfM library has a triangulator class, and we should reference (only) those that we actually referred to.

Reply:
Thanks @akshay-krishnan. We did not refer to this class when implementing it (I only recently found it), but I think it's a very useful reference to have closeby. The implementation is basically identical to ours.